edx_proctoring/admin.py

```diff
@@ -271,6 +271,7 @@ def student_username_for_review(self, obj):
         'exam_name_for_review',
         'student_username_for_review',
         'attempt_code',
+        'is_attempt_active',
```

Comment:
I'm also realizing that if we don't want to add a new field, we could also just display the info on the LMS Admin page. Not sure if it is better to actually add it to the model so it's also visible in snowflake or just add it here so that support can see it.

Reply:
I rather have this info recorded in the database because this way, we keep our business logic simple, and then make data query easier